media-tv/plex-media-server/plex-media-server-1.20.1-r1.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils systemd unpacker pax-utils
+
+MINOR_VERSION="3252-a78fef9a9"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}.${MINOR_VERSION}"
+
+URI="https://downloads.plex.tv/plex-media-server-new"
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+SRC_URI="
+	amd64? ( ${URI}/${_FULL_VERSION}/debian/plexmediaserver_${_FULL_VERSION}_amd64.deb )
+	x86? ( ${URI}/${_FULL_VERSION}/debian/plexmediaserver_${_FULL_VERSION}_i386.deb )
+"
+SLOT="0"
+LICENSE="Plex"
+RESTRICT="bindist strip mirror"
+KEYWORDS="-* ~amd64 ~x86"
+
+RDEPEND="
+	acct-group/plex
+	acct-user/plex
+	net-dns/avahi"
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/${_APPNAME}/.*"
+	"usr/lib/${_APPNAME}/Resources/Python/lib/python2.7/.*"
+)
+
+BINS_TO_PAX_MARK=(
+	"${ED}/usr/lib/plexmediaserver/Plex Script Host"
+	"${ED}/usr/lib/plexmediaserver/Plex Media Scanner"
+)
+
+S="${WORKDIR}"
+PATCHES=(
+	"${FILESDIR}/plexmediamanager.desktop.new.patch"
+	"${FILESDIR}/plexmediaserver.service.patch"
+)
+
+src_unpack() {
+	unpack_deb ${A}
+}
+
+src_install() {
+	# Remove Debian apt repo files
+	rm -r "etc/apt" || die
+
+	# Remove Debian specific files
+	rm -r "usr/share/doc" || die
+
+	# Copy main files over to image and preserve permissions so it is portable
+	cp -rp usr/ "${ED}"/ || die
+
+	# Make sure the logging directory is created
+	local logging_dir="/var/log/pms"
+	dodir "${logging_dir}"
+	fowners "${_USERNAME}":"${_USERNAME}" "${logging_dir}"
+	keepdir "${logging_dir}"
+
+	# Create default library folder with correct permissions
+	local default_library_dir="/var/lib/${_APPNAME}"
+	dodir "${default_library_dir}"
+	fowners "${_USERNAME}":"${_USERNAME}" "${default_library_dir}"
+	keepdir "${default_library_dir}"
+
+	# Install the OpenRC init/conf files
+	doinitd "${FILESDIR}/init.d/${PN}"
+	doconfd "${FILESDIR}/conf.d/${PN}"
+
+	# Mask Plex libraries so that revdep-rebuild doesn't try to rebuild them.
+	# Plex has its own precompiled libraries.
+	_mask_plex_libraries_revdep
+
+	# Install systemd service file
+	systemd_newunit "${FILESDIR}/systemd/${PN}.service" "${PN}.service"
```

Comment:
```
systemd_newunit "${ED}"/usr/lib/plexmediaserver/lib/plexmediaserver.service
```
We already apply a patch to fix PLEX_MEDIA_SERVER_INFO_PLATFORM_VERSION using the Gentoo Profile